cfme/services/catalogs/orchestration_template.py

```diff
@@ -237,7 +239,7 @@ class AddDialog(CFMENavigateStep):
     VIEW = AddDialogView
 
     def step(self):
-        self.view.configuration.item_select('Create Service Dialog from Orchestration Template')
+        cfg_btn("Create Service Dialog from Orchestration Template")
```

Comment:
This is the error when using the currently implemented methods.

    def move_to_element(self, locator, *args, **kwargs):
        """Moves the mouse cursor to the middle of the element represented by the locator.
    
            Can handle moving to the ``<option>`` tags or Firefox being pissy and thus making it utilize
            a JS workaround, ...
    
            Args: See :py:meth:`elements`
    
            Returns:
                :py:class:`selenium.webdriver.remote.webelement.WebElement`
            """
        self.logger.debug('move_to_element: %r', locator)
        el = self.element(locator, *args, **kwargs)
        if el.tag_name == "option":
            # Instead of option, let's move on its parent <select> if possible
            parent = self.element("..", parent=el)
            if parent.tag_name == "select":
                self.move_to_element(parent)
                return el
        move_to = ActionChains(self.selenium).move_to_element(el)
        try:
            move_to.perform()
        except MoveTargetOutOfBoundsException:
            # ff workaround
            self.execute_script("arguments[0].scrollIntoView();", el)
            try:
                move_to.perform()
            except MoveTargetOutOfBoundsException:  # This has become desperate now.
                raise MoveTargetOutOfBoundsException(
                    "Despite all the workarounds, scrolling to `{}` was unsuccessful.".format(
>                       locator))
E               MoveTargetOutOfBoundsException: Message: Despite all the workarounds, scrolling to `<selenium.webdriver.remote.webelement.WebElement (session="1dc853fb-1046-401f-87c2-788c93661a6a", element="47")>` was unsuccessful.

../lib/python2.7/site-packages/widgetastic/browser.py:373: MoveTargetOutOfBoundsException